M2/Macaulay2/packages/GKMManifolds/Documentations_GKMManifolds.m2

```diff
@@ -0,0 +1,2109 @@
+
```

Comment:
"Documentation" is already a collective noun and should not be made plural (in the name of this file)

Reply:
yes